extensions/amp-ad-network-doubleclick-impl/doubleclick-rtc.md

```diff
@@ -17,6 +17,15 @@ AMP Real Time Config (RTC) is a feature of Fast Fetch that allows Publishers to
 
 For instructions on how to set the rtc-config attribute on the amp-ad, refer to [Setting Up RTC-Config](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/rtc-publisher-implementation-guide.md#setting-up-rtc-config) in the Publisher Implementation Guide.
 
+## Available URL Macros
+Doubleclick's RTC implementation has made many macros available for RTC url expansion. Please note that the time to expand the URL is counted against the RTC timeout. Currently available macros are as follows:
+
+* **PAGEVIEWID**: pageViewId for the amp-ad element.
+* **HREF**: window.location.href
+* **TGT**: The JSON stringified targeting parameter extracted from the JSON attribute on the amp-ad element.
```

Comment:
Ditto

Reply:
No we do not, but we limit the URL as a whole.